collection_mutation.cc

```diff
@@ -166,10 +166,13 @@ collection_mutation_view_description::materialize(const abstract_type& type) con
     return m;
 }
 
-bool collection_mutation_description::compact_and_expire(column_id id, row_tombstone base_tomb, gc_clock::time_point query_time,
+compact_and_expire_result collection_mutation_description::compact_and_expire(column_id id, row_tombstone base_tomb, gc_clock::time_point query_time,
```

Comment:
I see why you added operator bool previously. I guess we can keep it, but even the bool return value is on shaky grounds.

Reply:
Yes, the operator bool is mainly needed for incremental patching. I can drop it at the end. I don't like it either.

Reply:
Dropped in v2.